docker-entrypoint.sh

```diff
@@ -55,11 +55,12 @@ VOLUME_DIR="/godata"
 # these 3 vars are used by `/go-agent/agent.sh`, so we export
 export AGENT_WORK_DIR="/go"
 export STDOUT_LOG_FILE="/go/go-agent-bootstrapper.out.log"
```

Comment:
No need to export this. It won't be made use of in https://github.com/gocd/gocd/blob/master/installers/go-agent/release/agent.sh#L126

Reply:
This needs to be `export`ed since otherwise the `agent.sh` will not actually see the file.